columnifier/parquet.go

```diff
@@ -65,36 +68,32 @@ func NewParquetColumnifier(st string, sf string, rt string, output string, confi
 }
 
 // Write reads, converts input binary data and write it to buffer.
-func (c *parquetColumnifier) Write(data []byte) (int, error) {
+func (c *parquetColumnifier) WriteFromReader(reader io.Reader) (int, error) {
 	// Intermediate record type is map[string]interface{}
-	c.w.MarshalFunc = parquet.MarshalMap
-	records, err := record.FormatToMap(data, c.schema, c.rt)
+	c.w.MarshalFunc = marshal.MarshalJSON
+	decoder, err := record.NewJsonDecoder(reader, c.schema, c.rt)
 	if err != nil {
 		return -1, err
 	}
 
 	beforeSize := c.w.Size
-	for _, r := range records {
-		if err := c.w.Write(r); err != nil {
+	for {
+		var v string
+		err = decoder.Decode(&v)
```

Comment:
Honestly, I consider what a better name is. I think `Decoder` reminds users something like `json.Decoder` and such struct's typically converts `[]byte` <-> go struct values or `interface{}` ... anyway I renamed it to `jsonStringConverter` to avoid such misleading. How about that?

Reply:
It looks better 👍